docs/ja/core-concepts/ngsiv2-vs-ngsild.md

```
@@ -108,28 +108,28 @@ interface EntityDocument {
+| NGSI-LD UPDATE/DELETE | 見つからない (404) | 許可される |
 
 ### レガシーエンティティ
 
-プロトコルアイソレーション導入以前に作成されたエンティティ (つまり、データベースに `protocol` フィールドがないもの) は `'ngsild'` として扱われます。これらは NGSI-LD API を介してのみアクセス可能です。
+プロトコル分離が導入される前に作成されたエンティティ(つまり、データベースに `protocol` フィールドがないもの)は、`'ngsild'` として扱われます。これらは NGSI-LD API 経由でのみアクセス可能です。
 
-### フェデレーション経由のクロスプロトコルアクセス
+### Federation によるプロトコル間アクセス
 
-直接的なクロスプロトコルアクセスはサポートされていません。プロトコル間でエンティティにアクセスする必要がある場合は、**フェデレーション** (Context Source Registration) を使用して、一方の GeonicDB インスタンスを他方のプロトコルのコンテキストプロバイダーとして登録してください。詳細は [フェデレーション](#federation) セクションを参照してください。
+直接的なプロトコル間アクセスはサポートされていません。プロトコル間でエンティティにアクセスする必要がある場合は、**Federation**(コンテキストソース登録)を使用して、ある GeonicDB インスタンスを他のプロトコルのコンテキストプロバイダーとして登録してください。詳細は [Federation](#federation) セクションを参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)